R/load_params.R

```diff
@@ -116,7 +116,8 @@ load_params = function(group = c("sleep", "metrics", "rawdata",
                           part5_agg2_60seconds = FALSE,
                           sensor.location = "wrist",
                           expand_tail_max_hours = NULL, recordingEndSleepHour = NULL,
-                          dataFormat = "raw", nonwear_approach = "2013")
+                          dataFormat = "raw", nonwear_approach = "2013",
+                          sep_reports = ",", sep_config = ",")
```

Comment:
Should we move `sep_reports` from `params_general` to `params_output` as it is specific to the GGIR output?

Also I see that there are a couple more arguments in `params_general` that could move to other categories:
- `sep_config` is also a type of output, by which it could fit in `params_output` even though it is later used as input.
- `nonwear_approach` may fit better with `params_cleaning`.

I think we should try to avoid having too many arguments in the general category as that category is more intended to have all arguments that do not fit with any of the other categories.